extensions/dicom-seg/src/components/BrushColorSelector.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const BrushColorSelector = ({ defaultColor, index, onNext, onPrev }) => (
+  <div>
+    <div
+      className="selector-active-segment"
+      style={{ backgroundColor: defaultColor }}
+    >
+      {index}
+    </div>
+    <div className="selector-buttons">
+      <button className="db-button" onClick={onPrev}>
+        Previous
+      </button>
+      <button className="db-button" onClick={onNext}>
```

Comment:
This is tricky. These components make the assumption that this class is set by the `viewer` project. A kind of soft dependency. We should either co-locate styles, or pull from a central/shared place like `@ohif/ui` in my opinion.

Reply:
If you do co-locate, they should be scoped to the component, or classes should use an extension prefix to avoid conflict. IE. `.dcmseg-button`

Reply:
Totally get it. Since we will not be using those outside this extension, I just scoped all styles within its corresponding extension component main style.

Reply:
Trying to see where these are now defined, and I can't see any pushed changes? Can you link me to the updated line?

Reply:
Nvm. I see how GitHub is filtering.